systemd/coreos-installer-reboot.service

```diff
@@ -2,7 +2,7 @@
 Description=Reboot after CoreOS Installer
 After=coreos-installer.service
 OnFailure=emergency.target
-OnFailureJobMode=isolate
+OnFailureJobMode=replace-irreversibly
```

Comment:
Huh weird, `replace-irreversibly` didn't work for me in the past (hence why the ignition-dracut services use `isolate`: https://github.com/coreos/ignition-dracut/blob/c4790bc283c8cc5318019ef8d5ad3929c8259d0c/dracut/30ignition/ignition-complete.target#L12). Though maybe it's time to retest this again and clean things up if it works again.

Reply:
Let me know what you find